tests/unit/cfg/aws_services_config

```diff
@@ -0,0 +1,10 @@
+[default]
+endpoint_url = https://localhost:1234/
+services = my-services
+
+[services my-services]
+s3 =
+  endpoint_url = https://localhost:5678/
+  addressing_style = path
```

Comment:
How does this reconcile against defining this in the default profile? I may have missed where that logic is handled.

Reply:
Nope shouldn't be there, need to remove that. Thanks!